lib/src/scss-shared/_nx-color-swatches.scss

```diff
@@ -111,6 +111,7 @@ $nx-red-300: #ff8286;
 $nx-red-200: #ff979a;
 $nx-red-100: #ffb6b8;
 $nx-red-50: #ffcbcc;
+$nx-red-25: #fff6f8;
```

Comment:
Shouldn't this be checked with @dturner32 ?

Reply:
> Shouldn't this be checked with @dturner32 ?

I confirmed all the colour changes with @dturner32. He came up with the colour above.